src/lib/relationship_graph.cc

```diff
@@ -25,9 +25,9 @@
 #include <boost/graph/connected_components.hpp>
 #include <boost/range/algorithm/find.hpp>
 
-#define DNG_GL_PREFIX "GL-"
-#define DNG_SM_PREFIX "SM-" // define also in newick.cc
-#define DNG_LB_PREFIX "LB-"
+#define DNG_GL_PREFIX "GL/"
+#define DNG_SM_PREFIX "SM/" // define also in newick.cc
+#define DNG_LB_PREFIX "LB/"
```

Comment:
Option 1: If we must use #define, then I suggest create a `constant.h` file.
Option 2: If we don't need to use #define, how about `extern const std::string .....` in *.h